com.unity.cinemachine/Runtime/Behaviours/CinemachineConfiner2D.cs

```diff
@@ -219,117 +217,112 @@ class VcamExtraState
         /// </summary>
         struct ShapeCache
         {
-            public ConfinerOven m_confinerOven;
-            public List<List<Vector2>> m_OriginalPath;  // in baked space, not including offset
+            public ConfinerOven confinerOven;
+            public List<List<Vector2>> originalPath;  // in baked space, not including offset
```

Comment:
We're supposed to be using PascalCase for publics
(not that it really matters - this is a private struct anyway, but it's good to get into the habit of doing it right, if we're going to be making these kinds of changes)